examples/norne_parameters.yml

```diff
@@ -4,48 +4,52 @@ flownet:
   data_source:
     eclipse_case: ../tests/data/norne/NORNE_ATW2013
     resample: M
+  pvt:
+    rsvd: ../tests/data/norne/rsvd.csv
```

Comment:
Put this file instead together with the other ones in `./examples/norne_static`?